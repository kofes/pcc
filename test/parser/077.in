function foo:integer;
begin
end;
procedure foo;
begin
end;
