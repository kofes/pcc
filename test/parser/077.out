Duplication of function "FOO" in pos (4, 11);
